omegaconf/omegaconf.py

```diff
@@ -579,7 +579,8 @@ def to_container(
         *,
         resolve: bool = False,
         enum_to_str: bool = False,
-        exclude_structured_configs: bool = False,
+        structured_config_mode: SCMode = SCMode.DICT,
+        exclude_structured_configs: Optional[bool] = None,
```

Comment:
Update the doc string to explain  structured_config_mode

Reply:
Done in https://github.com/omry/omegaconf/pull/552/commits/cb755cd6ab19fe7d700469d613a8b46e7b8b3506.